Clappr/Classes/Plugin/Playback/AVFoundation/AVFoundationNowPlayingBuilder.swift

```diff
@@ -0,0 +1,105 @@
+import MediaPlayer
+import AVKit
+import AVFoundation
+
+class AVFoundationNowPlayingBuilder: AVFoundationNowPlayingBuilderProtocol {
```

Comment:
We've called this `AVFoundationNowPlayingBuilder` and my first thought was that class would build a `AVFoundationNowPlaying` object or something like that. But this is not what's happening.

Can you help me understand what this class is building? Or should we rename it?

Reply:
Before [this commit](https://github.com/clappr/clappr-ios/pull/158/commits/d6ba0c613e0635b35a1d56d508d91767b1e3f6fd), the responsibility of this class was just built a list of items.
Now the responsibility of this class is:
* Build a list of items
* Set these items to the AVPlayerItem

Do you know a good name for this struct?